### PR TITLE
media-sound/audiotag: fix LICENSE, EAPI=7 bump

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This PR updates media-sound/audiotag for EAPI7 and fixes the LICENSE
Please review.

### CODE REVIEW

File: media-sound/audiotag/audiotag-0.19-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+EAPI=7
+
+DESCRIPTION="Command-line tool for mass tagging/renaming of audio files"
+HOMEPAGE="https://github.com/Daenyth/audiotag"
+SRC_URI="https://github.com/downloads/Daenyth/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="aac flac mp3 vorbis"
```
Comment (juippis) [inline media-sound/audiotag/audiotag-0.19-r1.ebuild]: Can these be moved to `optfeature` since they don't do anything in this ebuild?
(@gyakovlev)

Comment (mm1ke) [reply media-sound/audiotag/audiotag-0.19-r1.ebuild]: sounds reasonable, i've updated the ebuild to use ```optfeature``` instead.

**Comment by juippis** [comment]
```
has_version: Invalid atom: media-video/atomicparsley{-wez}
 * ERROR: media-sound/audiotag-0.19-r1::testworld failed (postinst phase):
 *   has_version: invalid atom: media-video/atomicparsley{-wez}
 *
 * Call stack:
 *          ebuild.sh, line 125:  Called pkg_postinst
 *        environment, line 548:  Called optfeature 'for m4a/mp4 support' 'media-video/atomicparsley{-wez}'
 *        environment, line 511:  Called has_version 'media-video/atomicparsley{-wez}'
 *   phase-helpers.sh, line 929:  Called ___best_version_and_has_version_common 'media-video/atomicparsley{-wez}'
 *   phase-helpers.sh, line 910:  Called die
 * The specific snippet of code:
 *                      die "${FUNCNAME[1]}: invalid atom: ${atom}"
 *
 * If you need support, post the output of `emerge --info '=media-sound/audiotag-0.19-r1::testworld'`,
 * the complete build log and the output of `emerge -pqv '=media-sound/audiotag-0.19-r1::testworld'`.
 * The complete build log is located at '/tmp/portage/media-sound/audiotag-0.19-r1/temp/build.log'.
 * The ebuild environment file is located at '/tmp/portage/media-sound/audiotag-0.19-r1/temp/environment'.
 * Working directory: '/tmp/portage/media-sound/audiotag-0.19-r1/homedir'
 * S: '/tmp/portage/media-sound/audiotag-0.19-r1/work/audiotag-0.19'
 * FAILED postinst: 1
>>> Regenerating /etc/ld.so.cache...

>>> Failed to execute postinst for media-sound/audiotag-0.19-r1
```
:(
FTR I thought it'd work too.

**Comment by mm1ke** [comment]
@juippis 

Nice catch.. Thx and fixed.

**Pull request closed without merging**